src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs

```diff
@@ -110,38 +114,46 @@ private static Hashtable IntrinsicTypeConverters
-                    temp[typeof(int)] = typeof(Int32Converter);
-                    temp[typeof(short)] = typeof(Int16Converter);
-                    temp[typeof(long)] = typeof(Int64Converter);
-                    temp[typeof(float)] = typeof(SingleConverter);
-                    temp[typeof(UInt16)] = typeof(UInt16Converter);
-                    temp[typeof(UInt32)] = typeof(UInt32Converter);
-                    temp[typeof(UInt64)] = typeof(UInt64Converter);
-                    temp[typeof(object)] = typeof(TypeConverter);
-                    temp[typeof(void)] = typeof(TypeConverter);
-                    temp[typeof(DateTime)] = typeof(DateTimeConverter);
-                    temp[typeof(DateTimeOffset)] = typeof(DateTimeOffsetConverter);
-                    temp[typeof(Decimal)] = typeof(DecimalConverter);
-                    temp[typeof(TimeSpan)] = typeof(TimeSpanConverter);
-                    temp[typeof(Guid)] = typeof(GuidConverter);
-                    temp[typeof(Array)] = typeof(ArrayConverter);
-                    temp[typeof(ICollection)] = typeof(CollectionConverter);
-                    temp[typeof(Enum)] = typeof(EnumConverter);
-                    temp[typeof(Uri)] = typeof(UriTypeConverter);
-
-                    // Special cases for things that are not bound to a specific type
-                    //
-                    temp[s_intrinsicNullableKey] = typeof(NullableConverter);
+                    Hashtable temp = new Hashtable
+                    {
+                        [typeof(bool)] = typeof(BooleanConverter),
+                        [typeof(byte)] = typeof(ByteConverter),
+                        [typeof(SByte)] = typeof(SByteConverter),
+                        [typeof(char)] = typeof(CharConverter),
+                        [typeof(double)] = typeof(DoubleConverter),
+                        [typeof(string)] = typeof(StringConverter),
+                        [typeof(int)] = typeof(Int32Converter),
+                        [typeof(short)] = typeof(Int16Converter),
+                        [typeof(long)] = typeof(Int64Converter),
+                        [typeof(float)] = typeof(SingleConverter),
+                        [typeof(UInt16)] = typeof(UInt16Converter),
+                        [typeof(UInt32)] = typeof(UInt32Converter),
+                        [typeof(UInt64)] = typeof(UInt64Converter),
+                        [typeof(object)] = typeof(TypeConverter),
+                        [typeof(void)] = typeof(TypeConverter),
+                        [typeof(DateTime)] = typeof(DateTimeConverter),
+                        [typeof(DateTimeOffset)] = typeof(DateTimeOffsetConverter),
+                        [typeof(Decimal)] = typeof(DecimalConverter),
+                        [typeof(TimeSpan)] = typeof(TimeSpanConverter),
+                        [typeof(Guid)] = typeof(GuidConverter),
+                        [typeof(Uri)] = typeof(UriTypeConverter),
+#if NETSTANDARD17
```

Comment:
Yes in general I like to avoid them. @alexperovich we should just remove the older build and so netstandard1.7 will be the default.